app/main.py

```diff
@@ -115,16 +120,29 @@ def grab_secrets(self):
                   with open(output_path, 'w') as cert_file:
                       cert_file.write(self.cert_to_pem(cert.cer))
 
+    def dump_secret(self, secret):
+              value = base64.decodestring(secret.value)
```

Comment:
what if the secret is not base64 encoded?

Reply:
moved to top level comments.